src/tools/cephfs/cephfs-shell

```diff
@@ -846,6 +846,10 @@ sub-directories, files')
                     self.perror('error: no such directory {} exists'.format(
                                 path.decode('utf-8')), end='\n',
                                 apply_style=True)
+                except libcephfs.ObjectNotEmpty:
```

Comment:
This block isn't reachable; it needs to be placed before `except libcephfs.Error`.